net.trowell.typesetter.json

```
@@ -0,0 +1,67 @@
+{
+    "id" : "net.trowell.typesetter",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "49",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "typesetter",
+    "finish-args" : [
+        "--device=dri",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "typesetter=debug"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "libspelling",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/libspelling.git",
+                    "tag": "0.4.8"
+                }
+            ]
+        },
+        {
+            "name" : "typesetter",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--buildtype=release"
+            ],
+            "post-install": [
+                "install -Dm644 data/fonts/Iosevka/*.ttf -t /app/share/fonts/",
+                "install -Dm644 typst.xml /app/share/mime/packages/$FLATPAK_ID.xml"
```

```suggestion
                "install -Dm644 ../data/fonts/Iosevka/*.ttf -t /app/share/fonts/",
                "install -Dm644 ../typst.xml /app/share/mime/packages/$FLATPAK_ID.xml"
```

since it relative to the build dir.